app/ota_client.py

```diff
@@ -573,7 +574,8 @@ def _set_statistics(self, sts):
                     staging_storage[f"elapsed_time_{_suffix}"] += int(
                         st.get("elapsed", 0) * 1000
                     )
-                    staging_storage[f"errors_{_suffix}"] = st.get("errors", 0)
+                    if _suffix == "download":
+                        staging_storage[f"errors_{_suffix}"] = st.get("errors", 0)
```

Comment:
increment operator should be used, right?

Reply:
Oh sorry I miss that!

Reply:
Oh sorry I miss that!